client/lib/signup/dependency-store.js

```diff
@@ -1,50 +1,38 @@
 /**
  * External dependencies
  */
-var store = require( 'store' ),
-	assign = require( 'lodash/assign' ),
-	keys = require( 'lodash/keys' ),
-	difference = require( 'lodash/difference' ),
-	isEmpty = require( 'lodash/isEmpty' );
+import keys from 'lodash/keys';
+import difference from 'lodash/difference';
+import isEmpty from 'lodash/isEmpty';
 
 /**
  * Internal dependencies
  */
-var Dispatcher = require( 'dispatcher' ),
-	config = require( 'config' ),
-	steps = require( 'signup/config/steps' ),
-	emitter = require( 'lib/mixins/emitter' );
+import {
+	SIGNUP_DEPENDENCY_STORE_RESET,
+	SIGNUP_DEPENDENCY_STORE_UPDATE_STATE,
+} from 'state/action-types';
 
-/**
- * Constants
- */
-const STORAGE_KEY = 'signupDependencies';
-
-/**
- * Module variables
- */
-var signupDependencies = {};
+import Dispatcher from 'dispatcher';
+import steps from 'signup/config/steps';
 
-var SignupDependencyStore = {
+const SignupDependencyStore = {
 	get: function() {
-		return signupDependencies;
+		return SignupDependencyStore.reduxStore.getState().signup.dependencyStore || {};
```

Comment:
We can create a selector for this which can also handle default values when the dependencyStore is not defined.
```
getSignupDependencyStore( SignupDependencyStore.reduxStore.getState() );
```